packages/lumx-react/src/components/side-navigation/SideNavigationItem.tsx

```diff
@@ -39,6 +39,9 @@ interface SideNavigationItemProps extends GenericProps {
     /** Whether or not the menu is selected. */
     isSelected?: boolean;
 
+    /** props that will be passed on to the Link */
+    linkProps?: Record<string, string>;
```

Comment:
We can be more precise and type this as `React.DetailedHTMLProps<React.AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>`.
It's the official type of html anchor elements. It's quite complicated, I don't know if it should be used or not but it would add intellisense on the linkProps